src/modules/CardsList.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="cards__products">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: 'CardsListing',
+})
+</script>
+
+<style scoped lang="scss">
+@import 'src/styles/widget.scss';
```

Comment:
I don't think you need this here? Also `<div class="cards">` should be moved from `CardsLogo` to this file. Think of each BEM block (e.g `.cards` or `.widget`) as a component